samples/kubeflow/README.md

```diff
@@ -53,6 +53,8 @@ cd samples/kubeflow/install/kubeflow
 
 cd -
 ```
+On an OpenShift 4+ cluster, follow the directions in this [README](https://github.ibm.com/the-mesh-for-data/samples/kubeflow/install/kubeflow/README.md) to install Kubeflow assuming Istio 1.7 is already installed on your cluster.
```

Comment:
Wrong link, change to github.com. Also I think you can use a relative path instead